addons/purchase/models/purchase.py

```diff
@@ -524,27 +404,6 @@ def _render_note_exception_quantity_po(order_exceptions):
             filtered_documents[(parent, responsible)] = rendering_context
         self.env['stock.picking']._log_activity(_render_note_exception_quantity_po, filtered_documents)
 
-    @api.multi
```

Comment:
Hello, the method _log_decrease_ordered_quantity just above and its sub method seems to have some references to stock pickings and moves